wizard/WizardCreateWallet2.qml

```diff
@@ -47,39 +98,318 @@ Rectangle {
+                    KeyNavigation.down: printPDFTemplate
+                    KeyNavigation.tab: printPDFTemplate
+                }
+
+                MoneroComponents.StandardButton {
+                    id: printPDFTemplate
+                    small: true
+                    text: qsTr("Print a template") + translationManager.emptyString
+                    tooltip: qsTr("Print a template to write down your seed") + translationManager.emptyString
+                    onClicked: {
+                        oshelper.openFile("wizard/template.pdf")
+                    }
+                    Accessible.role: Accessible.Button
+                    Accessible.name: qsTr("Print a blank template to write down your seed") + translationManager.emptyString
+                    KeyNavigation.up: copyToClipboardButton
+                    KeyNavigation.backtab: copyToClipboardButton
+                    KeyNavigation.down: walletCreationDate
+                    KeyNavigation.tab: walletCreationDate
+                }
+            }
+
+            RowLayout {
+                Layout.topMargin: 0
+                Layout.fillWidth: true
+                Layout.maximumWidth: seedListGridColumn.width
+                spacing: 10
+
+                ColumnLayout {
+                    spacing: 5
+                    Layout.fillWidth: true
+
+                    MoneroComponents.TextPlain {
+                        id: walletCreationDate
+                        font.family: MoneroComponents.Style.fontRegular.name
+                        font.pixelSize: 15
+                        font.bold: false
+                        textFormat: Text.RichText
+                        color: MoneroComponents.Style.dimmedFontColor
+                        text: qsTr("Creation date") + ": " + translationManager.emptyString
+                        themeTransition: false
+                        Accessible.role: Accessible.StaticText
+                        Accessible.name: qsTr("Creation date") + " " + walletCreationDateValue.text + translationManager.emptyString
+                        KeyNavigation.up: printPDFTemplate
+                        KeyNavigation.backtab: printPDFTemplate
+                        KeyNavigation.down: walletRestoreHeight
+                        KeyNavigation.tab: walletRestoreHeight
+                    }
+
+                    MoneroComponents.TextPlain {
+                        id: walletCreationDateValue
+                        property var locale: Qt.locale()
+                        property date currentDate: new Date()
+                        font.family: MoneroComponents.Style.fontRegular.name
+                        font.pixelSize: 16
+                        font.bold: true
+                        textFormat: Text.RichText
+                        color: MoneroComponents.Style.defaultFontColor
+                        text: currentDate.toLocaleDateString(locale, Locale.ShortFormat)
```

Comment:
What do you think about specifying it as `YYYY-MM-DD`? That's the format that is required for the restore height textbox.

Reply:
I prefer the default LocaleDateString. Later I want to display a calendar during wallet restore instead of asking for YYYY-MM-DD, which is not familiar to users from multiple countries.

Reply:
ALOT of beginners forget the "-" or just type the date incorrectly. This would save alot of people pain if a calendar was implemented :+1: